src/query.ts

```diff
@@ -1,2 +1,11 @@
-export { queryClientAtom } from './query/queryClientAtom'
+export { queryClientAtom, getQueryClientAtom } from './query/queryClientAtom'
```

Comment:
Would you later write some docs about the usage? (for both...)

Reply:
for sure :) also just to note I exported the getter because when using `queryClientAtom` it's type can be null, so this is better for clients to use